serverless.yml

```diff
@@ -48,6 +50,16 @@ custom:
     httpPort: ${env:API_PORT}
   pythonRequirements:
     layer: true
+  s3:
+    host: localhost
+    directory: /tmp
```

Comment:
Is this for testing?

Reply:
Yes! We need this to run serverless offline with s3 emulation. For now it's just nice to emulate and test things manually but when we implement integrations tests, this will be mandatory.